flink-container/docker/README.md

```diff
@@ -0,0 +1,44 @@
+# Apache Flink cluster deployment on docker using docker-compose
```

Comment:
Does this apply to both standalone and cluster mode? Want to get clarified since the PR title says it's for standaloneJobCluster